common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -937,6 +948,76 @@ def test_user_can_cancel_the_team_creation(self):
 
 
 @ddt.ddt
+class DeleteTeamTest(TeamFormActions):
+    """
+    Tests for deleting teams.
+    """
+
+    def setUp(self):
+        super(DeleteTeamTest, self).setUp()
+
+        self.set_team_configuration(
+            {'course_id': self.course_id, 'max_team_size': 10, 'topics': [self.topic]},
+            global_staff=True
+        )
+        self.create_edit_delete_team_page = CreateEditDeleteTeamPage(self.browser, self.course_id, self.topic)
+
+        self.team = self.create_teams(self.topic, num_teams=1)[0]
+        self.team_page = TeamPage(self.browser, self.course_id, team=self.team)
+        self.team_page.visit()
+
+    def test_cancel_delete(self):
```

Comment:
Yeah, Since they overlap so much I wasn't sure if we wanted to test each part of TNL-1913 and 1915 separately as they were made, or all together in one suite of tests.

I can add the types of tests being discussed here to https://github.com/edx/edx-platform/pull/9520, I agree that seems like a sensible place to test privileged access.